auth-api/config.py

```diff
@@ -103,6 +103,15 @@ class _Config(object):  # pylint: disable=too-few-public-methods
     # Config to skip migrations when alembic migrate is used
     SKIPPED_MIGRATIONS = ['authorizations_view']
 
+    # email server
```

Comment:
If you add new environment variables, you need to modify OpenShift pre-deploy and deploy templates as well.

Reply:
Done